benchmarks/csharp/Program.cs

```diff
@@ -3,16 +3,13 @@
 using System.Collections.Concurrent;
 using System.Diagnostics;
 using System.Text.Json;
-
 using CommandLine;
-
-using Glide;
-
 using LinqStatistics;
-
 using StackExchange.Redis;
+using Valkey.Glide;
+using static Valkey.Glide.ConnectionConfiguration;
 
-using static Glide.ConnectionConfiguration;
+namespace csharp_benchmark;
```

Comment:
Why sneaky case?

Reply:
Artifact of the project being named like that and a bulk namespace update happening.

The implicit, default namespace resulting is similar to the csproj name.

Reply:
Updated namespace